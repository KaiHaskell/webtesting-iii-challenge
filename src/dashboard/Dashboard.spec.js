// Test away
import React from "react";
import { render } from "@testing-library/react";
import Dashboard from "./Dashboard";

test("shows the controls and display", () => {
  const { getByText } = render(<Dashboard />);
  const display = getByText(/Open/i);
  const controls = getByText(/Close Gate/i);
  getByText(/Unlocked/i);

  expect(display).toBeDefined;
  expect(controls).toBeDefined;
  expect(display).toMatchSnapshot();
});
