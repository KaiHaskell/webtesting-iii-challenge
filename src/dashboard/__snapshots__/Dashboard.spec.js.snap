// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows the controls and display 1`] = `
<div
  class="led green-led"
>
  Open
</div>
`;
